GDR_environment.yaml

```diff
@@ -0,0 +1,17 @@
+name: GDR_environment
```

Comment:
This is a much cleaner environment, well done!

However, please do two things:

1. keep the file name `environment.yml`. Version control will ensure that we will be able to review Mark's first version
2. keep the environment name `gene_dependency_representations` (the original name) this will help tidy your conda environments and keep things consistent for the project.